app/views/games/index.html.erb

```diff
@@ -15,6 +15,10 @@
           <% end %>
 
           <div class="btn btn-primary">Join a Lobby</div>
+          <%= form_with url: join_instances_path, method: :post do |f| %>
+            <%= f.text_field :pin, value: params[:pin] %>
+            <%= f.submit "Search"%>
```

Comment:
Played with it and it works however this kinda gives us a third button now 😅 . I am not sure how we can turn it to just two buttons ideally the `create` and `join` buttons only. @hiroya210 any suggestions, hoping that you got something that would also make it easy for the front-end to manipulate.